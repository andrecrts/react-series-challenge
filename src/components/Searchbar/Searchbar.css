.searchbar {
  margin: 75px auto 0 auto;
  width: 75%;
}

.searchbar input[type=text] {
    padding: 6px;
    /* border: none; */
    margin-top: 8px;
    margin-right: 16px;
    font-size: 17px;
    width: 100%;
    height: 25px;
}
