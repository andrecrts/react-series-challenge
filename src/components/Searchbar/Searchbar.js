import React from 'react'
import './Searchbar.css'

const Searchbar = ({ handleKeyPress, handleChange }) => {

  return (
    <div className="searchbar">
      <input type="text"
        onKeyPress={handleKeyPress ? e => handleKeyPress(e) : null}
        onChange={handleChange ? e => handleChange(e) : null}/>
    </div>
  )
}

export default Searchbar
