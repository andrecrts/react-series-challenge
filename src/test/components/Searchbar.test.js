import React from 'react';
import Searchbar from '../components/Searchbar'
import renderer from 'react-test-renderer';

test('Searchbar snapshot', () => {
  const component = renderer.create(
    <Searchbar/>
  );
  let componentJSON = component.toJSON();
  expect(componentJSON).toMatchSnapshot();
})


