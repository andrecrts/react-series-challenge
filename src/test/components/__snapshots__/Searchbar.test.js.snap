// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Searchbar snapshot 1`] = `
<div
  className="sc-bdVaJa cpMCnx"
>
  <input
    className="sc-bwzfXH iYnflE"
    onChange={null}
    onKeyPress={null}
    placeholder="What are you looking for..."
    type="text"
  />
</div>
`;
